### PR TITLE
feat: print component id when spin up

### PR DESCRIPTION
**Pull request by lianghanzhen**
The available routes will be printed out in the following format:
`Component Id: URL (wildcard)`

` (wildcard)` will be printed out iff applicable.

Examples:
- hello: http://127.0.0.1:3000/hello
- static: http://127.0.0.1:3000/static (wildcard)

Signed-off-by: Hanzhen Liang <handrenliang@gmail.com>

### CODE REVIEW

**Comment by lianghanzhen** [comment]
Fix first point of #412

File: crates/manifest/src/lib.rs [hunk 1]
```diff
@@ -65,6 +65,8 @@ pub struct CoreComponent {
     /// ID of the component. Used at runtime to select between
     /// multiple components of the same application.
     pub id: String,
+    /// Description of the component(Optional).
```
Comment (itowlson) [inline crates/manifest/src/lib.rs:68]: Suggest removing `optional` here - it's already implicit in the `Option` type. 
```suggestion
    /// Description of the component.
```

**Review by itowlson — APPROVED** [review]
LGTM - thanks!

File: crates/manifest/src/lib.rs (hunk 1 above)
Comment (lianghanzhen) [reply crates/manifest/src/lib.rs]: Make sense. Resolved.

**Comment by radu-matei** [comment]
Could you also add a reference to the new field in the component configuration here, please?
https://github.com/fermyon/spin/blob/main/docs/content/configuration.md

Thanks!

**Pull request merged**